orderbook/src/api/filter.rs

```diff
@@ -59,6 +73,26 @@ pub mod test_util {
         assert_eq!(response_orders, orderbook_orders);
     }
     #[tokio::test]
+    async fn get_fee_info_() {
+        let filter = get_fee_info();
+        let sell_token: H160 = "6b175474e89094c44da98b954eedeac495271d0f".parse().unwrap();
```

Comment:
For the test it would be better to actually create the json body exactly as it is supposed to sent, probably with the serde_json::json macro. That way we see exactly what the expected encoding. The way the test is written now it is not clear how H160 will be serialized.